milestone3/frontend/src/components/home.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import recipes from '../recipeData';
+import './home.css'
+import RecipePreview from './recipePreview';
+
+export default function Home() {
+  return (
+    <main>
+      <div>
+        <h1 class="recipesTitle"> Recipes </h1>
+        {recipes.map(recipe => 
+        <RecipePreview
+          name={recipe.name}
+          desc={recipe.info}
+          image={recipe.image}
+          link = {recipe.link}
+        />)}
```

Comment:
Make sure to use the key prop whenever you use map to render a group of components. You can learn more about maps and keys here: https://reactjs.org/docs/lists-and-keys.html.